examples/counter/Makefile

```diff
@@ -1,8 +1,9 @@
 PORT=5001
 
 bundle:
-	(cd ./core && npm run build)
+	npm install
+	npm run build
 
 serve:
 	make bundle
-	(cd ./build && ../../../node_modules/.bin/http-server -p $(PORT))
+	(cd ./build && ../node_modules/.bin/http-server -p $(PORT))
```

Comment:
since `http-server` is locally installed in this example, may be not access it from parent directory any more.